### DISCUSSION TITLE
[Feature] Enable Favourits Mark and Faces on shared Library Photos

### DISCUSSION THREAD

**Discussion by helloworld1987** [immich-app/immich#11899] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Please explore the possibility to also heart photos and mark Faces on Photos accessed through partner sharing.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #4278. As requested, please use the search before opening a request.

**Comment by helloworld1987** [comment]
I searched for it and didn't find it. Sorry!
